pkg/services/object/put/remote.go

```diff
@@ -29,13 +29,18 @@ type remoteTarget struct {
 	obj *object.Object
 
 	clientCache *cache.ClientCache
+
+	clientOpts []client.Option
```

Comment:
This field is also should be set at `pkg/services/object/put/streamer.go:146`

Reply:
Fixed, thx.